INSTALL.linux.rst

```diff
@@ -157,7 +157,31 @@ Common configuration
 #. Install Docker Compose  on target machine according to https://docs.docker.com/compose/install/
    (known working: docker-compose version 1.29.2, build 5becea4c)
 
-Manual deployment
------------------
 
-#. TODO(dmu) HIGH: Describe "Manual deployment" section ( https://thenewboston.atlassian.net/browse/BC-70 )
+Configure continuous deployment
++++++++++++++++++++++++++++++++
+
+#. Prepare github personal access token aka PAT (not github password) - it will be needed to
+   access the node docker image. Creating a personal access token documentations:
+   https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
+
+#. Configure Github Settings
+
+    #. Create new environment (https://github.com/thenewboston-developers/Node/settings/environments)
+    #. Update "Deployment branches" section of recently created environment. Choice "Selected branches"
+       in drop-down menu.
+    #. Add new deployment branch rule. Set "Branch name pattern" value as "development".
+
+#. Create deploy ssh key on target machine::
+
+    # Use empty pass phrase
+    ssh-keygen -f ~/.ssh/github
+    cat ~/.ssh/github.pub >> ~/.ssh/authorized_keys
+
+#. Create github repository secrets::
+
+    NODE_CONTINUOUS_DEPLOYMENT_ENABLED=True
+    NODE_DEPLOY_SSH_KEY=<content of ~/.ssh/github>
+    NODE_DEPLOY_SSH_HOST=<IP-address or domain name of target machine>
+    NODE_DEPLOY_SSH_USER=<username that has the corresponding public in ~/authorized_keys>
+    NODE_GITHUB_TOKEN=<personal access token (PAT)>
```

Comment:
There is no need to configure `NODE_GITHUB_TOKEN` manually. See how it is done in thenewboston-node:

```
        env:
          GITHUB_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
```